docs/Version-Comparison.md

```diff
@@ -32,7 +32,7 @@ When in doubt, new users are advised to try out OTP2 and switch to OTP1 if they
 | Feature | OTP1 | OTP2 |
 |---------|------|------|
 | OSM street data | yes | yes |
-| GTFS transit data | yes | yes |
+| GTFS transit data | yes | yes[1] |
```

Comment:
Rather than a footnote I'd instead inline the comment here: `yes (frequencies.txt not yet supported)`